.travis.yml

```diff
@@ -18,17 +19,13 @@ matrix:
         - os: osx
           env: QT5=True
 before_install:
-   - . ${TRAVIS_BUILD_DIR}/.travis/${TRAVIS_OS_NAME}.${TARGET_OS}.before_install.sh
+    - . ${TRAVIS_BUILD_DIR}/.travis/before_install.sh
```

Comment:
Yep', so if you want it to fit in a single line I'll have to do something like: `chmod +x files && file`, here, using `bash file` looked better to me.